### PR TITLE
feat(client): add script and styles to make headers hyperlinks

### PR DESCRIPTION
**Pull request by nishilfaldu**
# Notes for blog or docs authors:

-   The `image: ` field in frontmatter of blogs should always be a `png` (`svg` will not work for social previews)
-   You must also duplicate the asset used in `image :` inside of the [public folder](https://github.com/tembo-io/website/tree/main/public)
-   Please write a `description: ` in the frontmatter of any new blog or doc
-   Read more [here](https://github.com/tembo-io/website?tab=readme-ov-file#writing-a-blog-post-%EF%B8%8F)

### CODE REVIEW

File: src/layouts/Layout.astro [hunk 1]
```diff
@@ -693,3 +712,21 @@ const {
 		renderDiagrams(graphs);
 	}
 </script>
+
+<script>
```
Comment (DarrenBaldwin07) [inline src/layouts/Layout.astro:716]: Can we conditionally attach/run this script only for blogs/docs? We do not want this script on pages like tembo.io/ or tembo.io/pricing for example

Comment (DarrenBaldwin07) [reply src/layouts/Layout.astro:716]: OK if not possible. Unsure if astro lets you do this

Comment (DarrenBaldwin07) [reply src/layouts/Layout.astro:716]: I just dont want this to apply to headings with ids on non docs/blog pages

Comment (nishilfaldu) [reply src/layouts/Layout.astro:717]: could not find an `astro` way. But another way could be load the script twice - once in docslayout and another in bloglayout or like I did now - only running the major chunk of script if the path includes `/docs` or `/blog`

**Pull request merged**